npm_and_yarn/lib/dependabot/npm_and_yarn/helpers.rb

```diff
@@ -16,6 +16,7 @@ module Helpers
         /^.*(?<error>The "yarn-path" option has been set \(in [^)]+\), but the specified location doesn't exist)/
 
       # NPM Version Constants
+      NPM_V10 = 10
```

Comment:
Is it necessary to define these constants here as integers and in [npm_and_yarn/lib/dependabot/npm_and_yarn/package_manager.rb](https://github.com/dependabot/dependabot-core/pull/10985/files#diff-0225e8b2ab72264915b6d1084d8e43bc7ac3cb017fb4a749e03a78204185a648) as strings?  Could they be defined in one location and business logic makes the necessary conversion of number<->string as needed?

Reply:
Eventually, we’ll need to refactor this to define all versions as strings instead of integers. However, this cleanup will take place after we finish removing all feature flags related to versioning and deprecate the old ecosystem metrics collection. For now, the specs require the method to return an integer, and this change is outside the scope of the current issue.